### PR TITLE
Reference topic get task by id

### PR DESCRIPTION
**Pull request by eapearce**
I hope this is just the reference topic and a link change on the main tasks topic.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ After your system is ready, these tutorials show you how to perform common tasks
 * [Change the due-date of a task _(coming soon)_](#tutorials)
 * [Delete a task _(coming soon)_](#tutorials)
 * [Get a user by name](tutorials/get-a-user-by-name)
+* [Searching for users](search-for-a-user.md)
```
Comment (rbwatson) [inline docs/index.md:47]:
```suggestion
* [Search for a user](search-for-a-user.md)
```

File: docs/tutorials/get-a-user-by-id.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    {{base_url}}/users
+```
+<!-- {% endraw %}) -->
+   
+6. To search for a specific ID, append a question mark (`?`) to `/id` in the URL, followed by the relevant query parameters, and then click **Send** to submit the `GET` request to the To-Do service.
+  
+      **Example Request:**      
+<!-- {% raw %} -->
+```
+      {{base_url}}/users?lid=1
```
Comment (rbwatson) [inline docs/tutorials/get-a-user-by-id.md:45]:
```suggestion
      {{base_url}}/users?id=1
```
Typo?

File: docs/tutorials/search-for-a-user.md
Comment (rbwatson) [inline docs/tutorials/search-for-a-user.md:1]: Is there a reason for the abbreviated format used in this tutorial?

**Review by rbwatson — CHANGES_REQUESTED** [review]
Please review my comments and let me know if this and #46 are related or need to be merged in any specific order.

Thanks!

**Comment by eapearce** [comment]
Thank you! After reviewing the files, it looks like I was unsuccessful in trying to move some changes to the new branch, so some of the links that support the the tutorial topic (pull request 46) are in pull request 47, and I don't think I successfully reverted all the changes when I tried separating one branch into two. Is there a way to diff pull request 46 and 47? 47 might have everything that 46 does, with some of the additional link changes to the overview topic to make things more visible. Otherwise, I think merging 46 first and then 47 would work too.

**Comment by rbwatson** [comment]
I rebased your pull request and merged #46 into this one, so I'll close #46.
Review these changes and address whatever comments remain, then when it looks ready to you, 
squash the commits and it should be ready to merge.

Thanks!

**Review by rbwatson — APPROVED** [review]
Rebased and squashed for merge

**Pull request merged**